### PR TITLE
Media Manager migration docs

### PR DESCRIPTION
**Pull request by dwalkr**
todo
- [x] https://github.com/tinacms/tinacms/pull/1494
- [x] can pass directory into cms.media.open per https://github.com/tinacms/tinacms/pull/1497/
- [x] image field interface and example updates
- [x] add NextGithubMedia store tip on configure custom app page

### CODE REVIEW

File: content/guides/nextjs/github/configure-custom-app.md [hunk 1]
```diff
@@ -32,15 +41,14 @@ export default class Site extends App {
...
-          clientId: process.env.GITHUB_CLIENT_ID,
-          baseRepoFullName: process.env.REPO_FULL_NAME, // e.g: tinacms/tinacms.org,
-        }),
+        github,
       },
       /**
-       * 3. Use the Sidebar and Toolbar
+       * 3. Register the Media Store
+       */
+      media: new GithubMediaStore(github),
```
Comment (DirtyF) [inline content/guides/nextjs/github/configure-custom-app.md:49]:
```suggestion
      media: new NextGithubMediaStore(github),
```

Why not use the NextJS Media Store here?

Comment (kendallstrautman) [reply content/guides/nextjs/github/configure-custom-app.md:49]: I'll add this as a tip!

File: content/blog/media-update.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
+---
+title: Adding a Media Manager to Tina
+date: '2020-10-02T10:20:39-04:00'
+author: Kendall
```
Comment (kendallstrautman) [inline content/blog/media-update.md:4]:
```suggestion
author: Kendall Strautman
```

File: content/blog/media-update.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
+---
+title: Adding a Media Manager to Tina
```
Comment (dwalkr) [inline content/blog/media-update.md:2]:
```suggestion
title: Introducing the TinaCMS Media Manager
```

File: content/blog/media-update.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+Along the way, we've made some significant changes to the media and image field APIs to improve the overall experience of working with media. This post outlines all of the new features and breaking changes.
+
+## Features & Improvements
+
+### Adding a media store
+
+Now that the interface for using media with the CMS is better understood, we've decided to streamline the API for adding a media store to the CMS. Where media stores were previously registered to `cms.media.store` in the CMS config, they should now just be registered to `cms.media`.
+
+Before
```
Comment (kendallstrautman) [inline content/blog/media-update.md:25]:
```suggestion
**Before**
```

File: content/blog/media-update.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+```tsx
+new TinaCMS({
+  media: {
+    store: new MyMediaStore(),
+  },
+})
+```
+
+After
```
Comment (kendallstrautman) [inline content/blog/media-update.md:35]:
```suggestion
**After**
```

**Review by kendallstrautman — APPROVED** [review]

**Pull request merged**